app/main.py

```
@@ -0,0 +1,174 @@
+                    "description": "WebSocket endpoint for Pipecat connections",
+                },
+            ],
+        }
+
+    return app
+
+
+def start_server(host: str = "0.0.0.0", port: int = 8766, local_dev: bool = False):
+    """Start the WebSocket server"""
+    # Global variables for ngrok URL tracking
+    NGROK_URLS = []
```

Ruff (0.8.2)

Redefinition of unused `NGROK_URLS` from line 16

(F811)